.github/workflows/pytest.yml

```diff
@@ -89,7 +89,7 @@ jobs:
       - name: run pytest on notebooks (append coverage)
         if: matrix.py == '3.8'
         run: |
-          pytest -n 2 --dist loadfile --nbval --current-env --cov-append tutorials/
+          pytest -n 2 --dist loadfile --nbval --current-env --cov-append ./doc/src/tutorials/
```

Comment:
good catch! Sorry I've missed that in #830

Reply:
I just wanted to get rid of the folder link (luckily it only broke pytest and no sphinx links :wink: )